_posts/2018-04-24-illinihackers-queue-dev-night-recap.markdown

```diff
@@ -1,17 +1,19 @@
-```
 ---
 layout: post
 title:  "The Return of the HackNight"
 date:   2018-04-24 12:00:00
 categories: recap
-tags: IlliniHackers, HackNight
+tags: IlliniHackers, HackNight, featured
 image: /assets/article_images/queue_dev_night/hacking2.jpg
 author: Rishi Desai
 ---
-```
+
+
 
 For the HackIllinois team, open source never stops. This past Tuesday, we continued our work in open source through an open source HackNight. HackNights are an opportunity for student developers at UIUC to work on open source projects with peers, professors, and lead engineers. For our first HackNight, we invited the developers of the [Illinois Queue](https://github.com/illinois/queue) team to teach and work with students to improve the Queue - one of Illinois's latest open source projects. The Queue is a campus wide tool used by classes to keep track of students waiting in line for office hours.
 
+## The Night
+
 Nathan Walters, Genna Helsel, and Professor Wade Fagen Ulmschneider from the Queue development team started off the night by giving a short tutorial on how to get started, and then it was off to the races as the students at the HackNight started claiming issues and making PRs. After just 2 hours of work, there were [5 Pull Requests](https://github.com/illinois/queue/pulls) made and [4 new Issues](https://github.com/illinois/queue/issues) logged for the project!
```

Comment:
Can you link to the specific PR's and issues that were made, in like a table?

PR name, person who made it (GitHub handle), and link to PR